docs/usage/configuration/config_documentation.md

```diff
@@ -3765,6 +3806,109 @@ redis:
   port: 6379
   password: <secret_password>
 ```
+---
+## Worker configuration for workers configuration
+These switches configure each worker them self with the a worker configuration file.
+
+Note also the configuration for the
+[main process above](#worker-configuration-for-main-process).
+
+The manual how to configure workers is described in detail in the
+[worker documentation](../../workers.md)
+
+---
+### `worker_app`
```

Comment:
Is this required? Does it have a default value?

Reply:
IMO it is required and there is no default. But I am not sure.
Ther is a related issue: https://github.com/matrix-org/synapse/issues/13644#issue-1352303831

Reply:
This is correct. No default.

Reply:
Looking at the source it looks like Synapse assumes it is the main process if this value is not provided (or if it is provided and set to `synapse.app.homeserver`).

Wouldn't object to us saying this is required, but I think the text is also fine as it is.